src/routes/docs/products/databases/collections/+page.markdoc

```diff
@@ -285,6 +285,8 @@ If an attribute must be populated in all documents, set it as `required`.
 If not, you may optionally set a default value.
 Additionally, decide if the attribute should be a single value or an array of values.
 
+If needed, you can change an attribute's name, default value, size (for strings), and whether it is required or not after creation.
```

Comment:
> We refer to it as attribute key, right? Would it be did to be consistent?

Updated

1. You can't, unless you truncate the data manually first
2. No, because we don't do any truncation
3. It's synchronous, because we only change table metadata

Added a note about decreasing size